src/didymus/pebble.py

```diff
@@ -0,0 +1,36 @@
+# imports
+
+class Pebble:
+    '''
+    Class for a single pebble object. Contains information
+    necessary to determine the pebble location within the core
+    and track its material composition and unique id number.
```

Comment:
```suggestion
    Class representing a TRISO fuel pebble. Contains information
    necessary to determine the pebble location within the core
    and track its material composition and unique id number.
```

Reply:
Definitely think representing is better than for here, but (at least tentatively) there's no reason this could be used with "dummy" graphite-only spheres, so I'm hesitant to specifically call them fuel pebbles.  Maybe just "Class representing a pebble"?